rmw_connext_cpp/CMakeLists.txt

```diff
@@ -42,7 +44,7 @@ if(WIN32)
     PRIVATE "ROSIDL_TYPESUPPORT_CONNEXT_CPP_BUILDING_DLL")
 endif()
 
-ament_export_libraries(rmw_connext_cpp ${Connext_LIBRARIES})
+ament_export_libraries(rmw_connext_cpp ${rmw_connext_shared_cpp_LIBRARIES})
```

Comment:
Instead of using:

* `ament_export_definitions`
* `ament_export_include_directories`
* `ament_export_libraries`

for `rmw_connext_shared_cpp_*` it should simply use:

* `ament_export_dependencies`